static/app/views/onboarding/targetedOnboarding/types.ts

```diff
@@ -5,11 +5,16 @@ export type StepData = {
   platform?: PlatformKey | null;
 };
 
+// Not sure if we need platform info to be passed down
 export type StepProps = {
   active: boolean;
+  addPlatform: (platform: PlatformKey) => void;
+  genSkipOnboardingLink: () => React.ReactNode;
   onComplete: () => void;
   orgId: string;
   organization: Organization;
+  platforms: PlatformKey[];
```

Comment:
If the currently selected list of platforms is a server-side state, then we do not have to put it in the props here. It'll be easier to let the individual steps manage this. For example, the welcome screen doesn't need this at all. The platform select screen only need to read on load and write on clicking "Next Step. The docs setup screen only need to read on load.

Reply:
@Neo-Zhixing sure, once we figure out how we're handling the server-side state we can swap out the state management, right now this is mostly just a placeholder until then